src/bootstrap/tool.rs

```diff
@@ -654,7 +654,7 @@ tool_extended!((self, builder),
     CargoClippy, clippy, "src/tools/clippy", "cargo-clippy", {};
     Clippy, clippy, "src/tools/clippy", "clippy-driver", {};
     Miri, miri, "src/tools/miri", "miri", {};
-    CargoMiri, miri, "src/tools/miri", "cargo-miri", {};
+    CargoMiri, miri, "src/tools/miri/cargo-miri", "cargo-miri", {}; // FIXME: path test does not work
```

Comment:
@Mark-Simulacrum any idea why `./x.py build src/tools/miri/cargo-miri` would fail despite this?

```
thread 'main' panicked at 'Error: no rules matched src/tools/miri/cargo-miri.', src/bootstrap/builder.rs:220:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
failed to run: /home/r/src/rust/rustc.2/build/bootstrap/debug/bootstrap build src/tools/miri/cargo-miri
```

Reply:
CargoMiri needs to be added https://github.com/rust-lang/rust/blob/45127211566c53bac386b66909a830649182ab7a/src/bootstrap/builder.rs#L350 -- looks like CargoClippy is also missing?

Reply:
Ah yes that does it, thanks. :)